src/global-messages/types.ts

```diff
@@ -23,6 +23,7 @@ export type GlobalMessageRaw = {
   active: boolean;
   title?: LanguageAndTextType[];
   body: LanguageAndTextType[];
+  link?: string;
```

Comment:
May have different link urls for different languages, so Firestore should support that.